pkg/users/account-create-dialog.js

```diff
@@ -43,27 +43,27 @@ function AccountCreateBody({ state, errors, change }) {
     return (
         <Form isHorizontal onSubmit={apply_modal_dialog}>
             <FormGroup label={_("Full name")}
-                       helperTextInvalid={errors && errors.real_name}
-                       validated={(errors && errors.real_name) ? "error" : "default"}
+                       helperTextInvalid={errors?.real_name}
+                       validated={(errors?.real_name) ? "error" : "default"}
                        fieldId="accounts-create-real-name">
                 <TextInput id="accounts-create-real-name"
-                           validated={(errors && errors.real_name) ? "error" : "default"}
+                           validated={(errors?.real_name) ? "error" : "default"}
                            value={real_name} onChange={value => change("real_name", value)} />
             </FormGroup>
 
             <FormGroup label={_("User name")}
-                       helperTextInvalid={errors && errors.user_name}
-                       validated={(errors && errors.user_name) ? "error" : "default"}
+                       helperTextInvalid={errors?.user_name}
+                       validated={(errors?.user_name) ? "error" : "default"}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18448-20230311-055523-7e3e6fe9-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)